ebms-async/src/test/kotlin/no/nav/emottak/ebms/async/kafka/KafkaIntegrationTest.kt

```diff
@@ -26,10 +29,15 @@ class KafkaIntegrationTest {
     val kafkaConfig = config()
 
     fun noLocalKafkaEnv(): Boolean {
```

Comment:
En liten forenkling du kan vurdere her: Hva med å kalle denne `hasLocalKafkaEnv` i stedet for `noLocalKafkaEnv`? Det gjør det litt enklere å lese, siden vi slipper å tenke i det negative. :sweat_smile: